Ifzan/bonus.js

```diff
@@ -0,0 +1,26 @@
+class Employee {
+    constructor(name, age, gender, salary, date_of_join, address){
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.salary = salary;
+        this.date_of_join = date_of_join;
+        this.address = address;
+    }
+}
+
+class Department extends Employee{
```

Comment:
department is not an Employee => it  fails the Is-A test

Department can have employees => Department has list of employees can be said though